lib/src/rules/avoid_positional_boolean_parameters.dart

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.rules.avoid_positional_boolean_parameters;
+
+import 'package:analyzer/analyzer.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/util/dart_type_utilities.dart';
+
+const _desc = r'Avoid positional boolean parameters.';
+
+const _details = r'''
+
+**AVOID** positional boolean parameters.
+
+**BAD:**
+```
+new Task(true);
+new Task(false);
+new ListBox(false, true, true);
+new Button(false);
+```
+
+**GOOD:**
+```
+new Task.oneShot();
+new Task.repeating();
+new ListBox(scroll: true, showScrollbars: true);
+new Button(ButtonState.enabled);
+```
+
+''';
+
+bool _hasInheritedMethod(MethodDeclaration node) {
```

Comment:
This method appears in another rule (or more). Perhaps in a separate CL move them into `DartTypeUtilities`.